example_projects/transfer/nlb_observability_stack/variables.tf

```diff
@@ -0,0 +1,153 @@
+# AWS resource needed variables
+variable "tags" {
+  type = map(string)
+  default = {
+    Example = "test"
+  }
+  description = "Tags for AWS resources"
+}
+
+variable "bucket_name" {
+  type        = string
+  default     = "double-cloud-nlb-logs-bucket"
+  description = "Name of the sample bucket where the NLB logs should be stored"
+}
+
+variable "bucket_prefix" {
+  type        = string
+  default     = "metrics"
+  description = "Bucket prefix for NLB logs in S3 bucket"
+}
+
+variable "nlb_name" {
+  type        = string
+  default     = "preprod-metric-exp-dt-tls"
+  description = "Name of the load balancer to enable acces logs for"
+}
+
+variable "sqs_name" {
+  type        = string
+  default     = "s3-nlb-logs-queue"
+  description = "Name of the AWS SQS where the bucktect creation events should be sent to"
+}
+
+variable "aws_profile" {
+  type        = string
+  default     = "your-profile"
+  description = "AWS profile to use for API calls"
+}
+
+variable "region" {
+  type        = string
+  default     = "eu-central-1"
+  description = "Region where to create S3 bucket"
+}
+
+variable "vpc_id" {
+  type        = string
+  default     = "example"
+  description = "ID of the VPC the NLB is tied to"
+}
+
+variable "fqdn" {
+  type        = string
+  default     = "test"
+  description = "DNS name associated with the NLB"
+}
+
+variable "subnet_ids" {
+  type = list(string)
+  default = ["test"]
+  description = "Subnet ID's for NLB"
+}
+
+# DC resource needed variables
+variable "project_id" {
+  type        = string
+  default     = "example"
+  description = "ID of the DoubleCloud project in which to create resources"
+}
+
+variable "service_account_endpoint" {
```

Comment:
don't need `service_account_endpoint` and `service_account_token_endpoint`

P.S. default values are invalid

Reply:
defaults are now null

Reply:
removed